### PR TITLE
Colloscope

### PR DESCRIPTION
**Pull request by flashonfire**
Introducing Colloscope integration for polytech students!

This is the v1 of the polytech colloscope

Currently not working if you don't go to the tomuss tab before colloscope tab as it relies on tomuss data to get the student name.
~~Also missing a proper icon.~~

We will also need to publish the polytechcolloscopeclient lib to be able to build without melos

i will probably redesign the page in the future but i think it's fine as a first implementation

### CODE REVIEW

**Comment by hatch01** [comment]
No notifications ?

File: packages/polytechcolloscopeclient/pubspec.yaml [hunk 1]
```diff
@@ -4,18 +4,23 @@ version: 1.0.0
 # repository: https://github.com/my_org/my_repo
 
 environment:
-  sdk: ^3.1.3
+  sdk: '>=3.0.0 <4.0.0'
 
 # Add regular dependencies here.
 dependencies:
   beautiful_soup_dart: ^0.3.0
-  collection: ^1.18.0
```
Comment (hatch01) [inline packages/polytechcolloscopeclient/pubspec.yaml:12]: Euuu

**Comment by flashonfire** [comment]
> No notifications ?

Not yet

**Review by hatch01 — APPROVED** [review]
seems realy clean !

**Pull request merged**